src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -3193,6 +3369,34 @@ def set_up_aad_profile(self, mc) -> ManagedCluster:
         mc.aad_profile = aad_profile
         return mc
 
+    def set_up_api_server_access_profile(self, mc: ManagedCluster) -> ManagedCluster:
+        """Set up api server access profile and fqdn subdomain for the ManagedCluster object.
```

Comment:
suggest to put the unit test coverage data for the PR.

Reply:
Sure, updated in the description.

Unit test result:
- "covered_lines": 949,
- "num_statements": 1008,
- "percent_covered": 94.14682539682539,
- "missing_lines": 59,
- "excluded_lines": 0